modules/references/client/components/create-reference/Feedback.js

```diff
@@ -43,5 +68,7 @@ export default function Feedback({ feedback, onChangeFeedback }) {
 
 Feedback.propTypes = {
   feedback: PropTypes.string.isRequired,
+  recommend: PropTypes.string,
```

Comment:
maybe `recommend` could be a boolean with "unknown" state being `undefined` because it feels like recommendation has boolean semantics at the moment with "don't know" being just an absence of value

but I guess it's also fine to leave it as it is.

Reply:
we discussed changing this in the future if I remember correctly.